src/include/migraphx/autocast_fp8.hpp

```diff
@@ -37,7 +37,6 @@ This pass will convert model with fp8 input parameter to model with fp32
 input parameter and internally add casts to fp8 for those converted params.*/
 struct MIGRAPHX_EXPORT autocast_fp8_pass
 {
-    std::set<shape::type_t> fp8_types = {migraphx::shape::fp8e4m3fnuz_type};
```

Comment:
So with this the assumption then is that we want to always be explicit in the type so we can do the correct conversion instead of relying on autocast?

Reply:
No, it uses the fp8_types struct I created instead of this